contracts/Tokens/VTokens/VToken.sol

```diff
@@ -1496,11 +1496,6 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
             return fail(Error.MARKET_NOT_FRESH, FailureInfo.REDUCE_RESERVES_FRESH_CHECK);
         }
 
-        // Fail gracefully if protocol has insufficient underlying cash
-        if (getCashPrior() < reduceAmount) {
-            return fail(Error.TOKEN_INSUFFICIENT_CASH, FailureInfo.REDUCE_RESERVES_CASH_NOT_AVAILABLE);
-        }
-
```

Comment:
I would keep this check. `_reduceReservesFresh` can be invoked "externally" from `_reduceReserves`, where no check compared with the available cash is done

Reply:
This check is what causes `accrueInterest` to fail. Should we have another one `_reduceFreshReserveSafe` that can be executed in situations outside of `accrueInterest` and includes this check?

Reply:
But now we are not invoking `_reduceFreshReserve` with a `reduceAmount` greater than `getCashPrior()`, so `accrueInterest` shouldn't fail anymore

Reply:
That's right